src/core/Track.cpp

```diff
@@ -4,7 +4,11 @@
 lif::Track lif::getNthTrack(int num, float start, float length) {
 	using lif::DIRSEP;
 	std::stringstream ss;
+#if defined(SFML_SYSTEM_MACOS)
```

Comment:
I think you can remove the ifdef here and just use assetDir in all cases

Reply:
Will do